addons/sms/tools/sms_tools.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+
+from odoo.tools import html_keep_url, misc
+
+
+def render_sms_notification(text):
```

Comment:
``sms_content_to_rendered_html`` ? something implying we generate html, even if it is a simple one just for links ?